bedrock/mozorg/templates/mozorg/about/leadership.html

```diff
@@ -1289,6 +1283,13 @@ <h3 class="group-title">{{ _('Former Board Members') }}</h3>
           <figcaption><h4 class="fn" itemprop="name">Ronaldo Lemos</h4></figcaption>
         </figure>
       </li>
+
+      <li class="vcard" itemscope itemtype="http://schema.org/Person">
```

Comment:
Nit: They're sorted alphabetically by surname, so she should go after Davidson, before Hoffman.